.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### What is the issue
+...
+
+### What does this PR fix and why was it fixed
+...
+
+### Checklist before you submit for review
+- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version
+- [ ] Use `NoSpamLogger` for log lines that may appear frequently in the logs
+- [ ] Verify test results on Butler
+- [ ] Test coverage for new/modified code is > 80%
+- [ ] Proper code formatting
+- [ ] Proper title for each commit staring with the project-issue number, like CNDB-1234
+- [ ] Each commit has a meaningful description
+- [ ] Each commit is not very long and contains related changes
+- [ ] Renames, moves and reformatting are in distinct commits
```

Comment:
Same as above: sometimes it's just artificial or inconvenient to split stuff out.

Reply:
I disagree here - we need to think about the reviewers - renaming a method, class or reformatting the code along with other changes in a single commit makes the review a way more difficult. It is easy to miss the important changes. When reformatting / renamings are isolated in their own commits, the reviewer can simply skip them and focus on real changes, which make the review faster and better